components/automate-ui/src/app/services/projects-filter/projects-filter.actions.ts

```diff
@@ -0,0 +1,35 @@
+import { Action } from '@ngrx/store';
+import { HttpErrorResponse } from '@angular/common/http';
+import { ProjectsFilterOption } from './projects-filter.reducer';
+
+export enum ProjectsFilterActionTypes {
```

Comment:
I was scratching my head at first trying to figure out what "options" this refers to throughout the code... It would help a lot, I think, to globally say "project" instead of "option".

Reply:
Dang, [`ProjectsFilterOption`](https://github.com/chef/automate/pull/172/files/e0dbabdeabf6a4c8cf7ba83aa3bd755a62b16a21#diff-b65e56f9fdc34d14ce2d28451cefeccaR8) is what I had intended to be in your head. I didn't go with just `Project` since that's a separate entity type not to be confused with. I originally had `LOAD_PROJECTS_FILTER_OPTIONS` but `ProjectsFilter.LoadProjectsFilterOptions` felt redundant compared to `ProjectsFilter.LoadOptions`.

Reply:
Probably worth getting a consensus on this before changing it.

Reply:
I'm alright with `options` as it differentiates them from project management calls to the Projects API. Adding `ProjectsFilterOption` where possible makes it more clear